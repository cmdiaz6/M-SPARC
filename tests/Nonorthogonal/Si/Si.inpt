# Test: Si8 #
LATVEC:
0.5 0.5 0.0
0.0 0.5 0.5
0.5 0.0 0.5
CELL: 10.2 10.2 10.2
FD_GRID: 20 20 20
KPOINT_GRID: 2 2 2
FD_ORDER: 12
BC: P P P
EXCHANGE_CORRELATION: LDA_PW
TOL_SCF: 1e-6
TOL_POISSON: 1e-8
TOL_PSEUDOCHARGE: 1e-8
ELEC_TEMP: 315.773
ELEC_TEMP_TYPE: fd
NSTATES: 25
PRINT_FORCES: 1
PRINT_ATOMS: 1
#MIXING_VARIABLE: potential
#MIXING_PRECOND: kerker
