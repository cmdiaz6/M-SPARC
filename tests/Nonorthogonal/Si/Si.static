***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of atoms:
      0.0000000000       0.0000000000       0.0000000000
      0.5000000000       0.5000000000       0.0000000000
      0.5000000000       0.0000000000       0.5000000000
      0.0000000000       0.5000000000       0.5000000000
      0.2500000000       0.2500000000       0.2500000000
      0.7500000000       0.7500000000       0.2500000000
      0.7500000000       0.2500000000       0.7500000000
      0.2500000000       0.7500000000       0.7500000000
Atomic forces (Ha/Bohr):
      0.1634526826       0.1634532218       0.1634533934
      0.1634531145       0.1634532215       0.1634528212
      0.1634530867       0.1634533955       0.1634533201
      0.1634530384       0.1634526518       0.1634531184
     -0.1634534116      -0.1634531235      -0.1634530188
     -0.1634528105      -0.1634531573      -0.1634535025
     -0.1634528834      -0.1634527059      -0.1634529827
     -0.1634528167      -0.1634535039      -0.1634531491
