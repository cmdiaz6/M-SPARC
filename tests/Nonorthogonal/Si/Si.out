***************************************************************************
*                      M-SPARC v1.0.0 (Feb 03, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 01-Feb-2020  Start time: 23:53:05                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.200000 10.200000 10.200000 
LATVEC:
0.500000000000000 0.500000000000000 0.000000000000000 
0.000000000000000 0.500000000000000 0.500000000000000 
0.500000000000000 0.000000000000000 0.500000000000000 
FD_GRID: 20 20 20
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 2 2 2
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.000999993
CHEB_DEGREE: 17
NSTATES: 25
EXCHANGE_CORRELATION: LDA_PW
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: Nonorthogonal/Si/Si.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.510000 (Bohr)
Number of symmetry adapted k-points:  4
Output printed to                  :  Nonorthogonal/Si/Si.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si_old.psp
lloc                               :  4
Pseudocharge radii of atom type 1  :  9.69 9.69 9.69
Number of atoms of type 1          :  8
Estimated total memory usage       :  71.17 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.8230799833E+00        1.478E-01        25.095
2            -3.8477254460E+00        5.432E-02        7.324
3            -3.8514445937E+00        1.685E-02        7.239
4            -3.8519526052E+00        3.715E-03        7.269
5            -3.8519755737E+00        1.994E-03        7.184
6            -3.8519831056E+00        1.655E-03        7.259
7            -3.8519829754E+00        1.536E-03        7.135
8            -3.8519753225E+00        2.552E-03        7.195
9            -3.8519683611E+00        3.325E-03        7.176
10           -3.8519873404E+00        3.287E-04        7.143
11           -3.8519874893E+00        4.185E-05        7.097
12           -3.8519874894E+00        3.297E-05        7.464
13           -3.8519874913E+00        3.810E-06        7.102
14           -3.8519874804E+00        7.280E-06        6.975
15           -3.8519874988E+00        3.167E-06        6.991
16           -3.8519874866E+00        3.404E-07        6.838
Total number of SCF: 16    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.8519874866E+00 (Ha/atom)
Total free energy                  : -3.0815899893E+01 (Ha)
Band structure energy              :  3.8212168927E+00 (Ha)
Exchange correlation energy        : -1.0622388960E+01 (Ha)
Self and correction energy         : -1.0227200710E+02 (Ha)
Entropy*kb*T                       : -1.4839235247E-03 (Ha)
Fermi level                        :  4.0726126009E-01 (Ha)
Average force                      :  2.8310905449E-01 (Ha/Bohr)
Maximum force                      :  2.8310936389E-01 (Ha/Bohr)
Time for force calculation         :  1.706 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  143.896 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
